generator/generator.go

```diff
@@ -327,6 +328,7 @@ func main() {
 				ShortenEnumLeafNames:                 *shortenEnumLeafNames,
 				EnumOrgPrefixesToTrim:                enumOrgPrefixesToTrim,
 				UseDefiningModuleForTypedefEnumNames: *useDefiningModuleForTypedefEnumNames,
+				IncludeDescriptions:                  *includeDescriptions,
```

Comment:
Should this just be in `GoOpts` since I'm not sure that it is applicable in other contexts right? Alternatively it could be in the `GeneratorConfig` directly since that's where we have `GenerateJSONSchema` specified. I don't think it is a transformation option.

Reply:
Yes, I think grouping it with `GenerateJSONSchema` under `GeneratorConfig` certainly makes sense.